components/Albums.js

```diff
@@ -0,0 +1,26 @@
+import data from '../albums.json';
+import React, { useEffect, useState } from 'react';
+import { View, Text } from 'react-native';
+
+const Albums = () => {
+	const [album, setAlbum] = useState({});
+	console.log(data.items[2].name);
+
+	const generateAlbum = () => {
+		fetch('../albums.json')
+			.then((res) => res.json())
+			.then((data) => setAlbum(data));
+	};
+
+	useEffect(() => {
+		generateAlbum();
+	});
+
+	return (
+		<View>
+			<Text>{JSON.stringify(data.items[2].name)} </Text>
+		</View>
+	);
+};
+
+export default Albums;
```

Comment:
I might have missed this from the classes since we did not do the same project, but I can't see it imported somewhere so just wanted to comment in case it is not used :)